.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!--- Please always add a PR description as if nobody knows anything about the context these changes come from. -->
+<!--- Even if we are all from our internal team, we may not be on the same page. -->
+<!--- Write this PR as you were contributing to a public OSS project, where nobody knows you and you have to earn their trust. -->
+<!--- This will improve our projects in the long run! Thanks. -->
+
+#### Description
+<!--- Describe your changes in detail -->
+
+#### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+#### How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, tests ran to see how -->
+<!--- your change affects other areas of the code, etc. -->
```

Comment:
This section means that we should update PR's title once we have tested our changes. Normally we open PR's just with the Description and the context. I was wondering if the test docs can be moved away from PR's description.

Reply:
Ok maybe it's too much writing for the intended effort.
What I intended here was "just drop a line of how did you test this". That could be:
* execute http request in local environment/io-mock
* I wrote extensive tests
* nothing